content/vue/step02.md

```diff
@@ -0,0 +1,124 @@
+
+We also need to add the akryum:vue-component package to allow us to create files with the .vue extension, which makes it possible to create Vue components in the single file format.
+
+```sh
+meteor add akryum:vue-component
+```
+
+### Replace `blaze-html-templates` with `static-html`
+
+By default, new Meteor applications use Blaze as their templating engine. While it's possible for a Meteor application to use Blaze and Vue simultaneously, the application we're building in this tutorial does not need Blaze at all. Since you created your Meteor application with `meteor create` the `blaze-html-templates` package is inlcuded, we recommend that you remove support for Blaze and add support for static HTML templates.
+
+To prevent processing `.html` files as Blaze templates, first remove the `blaze-html-templates` package:
+
+```sh
+meteor remove blaze-html-templates
+```
+
+Now, to ensure `.html` files are processed as static HTML, add the `static-html` package:
+
+```sh
+meteor add static-html
+```
+
+The `static-html` package is not specific to Vue. It simply turns `<head>` and `<body>` fragments found in `.html` files into raw HTML that will be served from the Meteor web server. Later, your Vue application will render its components into this HTML.
+
+Note that both `blaze-html-templates` and `static-html` are _Meteor packages_, rather than npm packages, because they need to register _compiler plugins_ that determine how `.html` files are processed. Controlling compilation is one of several key features that make Meteor packages more powerful than npm packages.
+
+### Replace the starter code
+
+To get started, let's replace the code of the default starter app. Then we'll talk about what it does.
+
+First, replace the content of the **`client/main.html`** file:
+
+{{> DiffBox tutorialName="simple-todos-vue" step="2.2"}}
+
+Second, replace the contents of the **`client/main.js`** module:
+
+{{> DiffBox tutorialName="simple-todos-vue" step="2.3"}}
+
+Now we need to create a new directory called `imports`, with a directory called `ui` inside of it. There will soon be other directories besides `ui` within `imports`, but we'll start with `ui`.
+
+> Note: in previous versions of Meteor, the `imports` directory was special because files outside the `imports` directory were loaded automatically when the application started, whereas files inside the `imports` directory were only loaded when imported using an `import` declaration or a `require` statement. As of Meteor 1.7, the entry point for both client and server JavaScript is determined by the `meteor.mainModule` section in `package.json`. In other words, as far as JavaScript code is concerned, the entire application now behaves as if it was inside an `imports` directory, so you don't need to worry as much about the `imports` directory now.
```

Comment:
This is not quite true ... Starting off with Meteor 1.7, the system is checking for the existance of the `meteor.mainModule` section in `package.json`. If it exists, this is used as the entry point and everything has to be imported. If this section is missing (even on Meteor 1.7+), it will import everything eagerly excepted for the files inside the `import` folder.

In my test project, which I started running `meteor create --bare` this section was not present, means that everything outside my `imports` folder is eagerly loaded - despite of that I'm working on Meteor `1.9.2`.